homeassistant/components/namecheapdns/__init__.py

```diff
@@ -19,33 +20,52 @@
 
 UPDATE_URL = 'https://dynamicdns.park-your-domain.com/update'
 
+DEFAULT_HOST = '@'
+
 CONFIG_SCHEMA = vol.Schema({
     DOMAIN: vol.Schema({
         vol.Required(CONF_DOMAIN): cv.string,
         vol.Required(CONF_PASSWORD): cv.string,
-        vol.Optional(CONF_HOST, default='@'): cv.string,
-    })
+        vol.Optional(CONF_HOST, default=DEFAULT_HOST):
+        cv.string,
+        vol.Optional(CONF_HOSTS): cv.ensure_list,
+    }),
+    cv.deprecated(
+        CONF_HOST,
+        replacement_key=CONF_HOSTS,
+        invalidation_version='0.91'),
+    _ensure_list('hosts')
 }, extra=vol.ALLOW_EXTRA)
 
 
+def _ensure_list(key: str):
```

Comment:
Why do we need this validator? Why can't we use `cv.ensure_list` directly?

Reply:
See this thread with examples: https://github.com/home-assistant/home-assistant/pull/21177#discussion_r257807140

Reply:
Ok. I suggest we avoid this entirely with my suggestion above.